src/main/resources/META-INF/rewrite/java-version-17.yml

```diff
@@ -32,6 +32,7 @@ recipeList:
   - org.openrewrite.java.migrate.lang.StringFormatted
   - org.openrewrite.java.migrate.lombok.UpdateLombokToJava17
   - org.openrewrite.github.SetupJavaUpgradeJavaVersion
+  - org.openrewrite.java.InstanceOfPatternMatch
```

Comment:
@timtebeek Thanks. Somehow I grabbed the wrong recipe.

Reply:
No worries there; it's easily overlooked; as a side effect we tested the CI and integration prior to a release of openrewrite/rewrite, so we're all clear here.

Would you want to [wait for a response on the potential cleanup package move](https://github.com/openrewrite/rewrite/pull/2690#issuecomment-1410362831) before we merge?

Reply:
Yes. Let's wait for that. Also, I am unsure if the new recipe is already available in the releases we use in the CI builds.

Reply:
Great; approved already; will monitor and merge, unless someone else beats me to it. Thanks again!